utils/isValidLCDate.js

```diff
@@ -0,0 +1,29 @@
+const dayjs = require('dayjs');
+let isBetween = require('dayjs/plugin/isBetween');
+let utc = require('dayjs/plugin/utc');
+dayjs.extend(isBetween);
+dayjs.extend(utc);
+
+function isValidLCDate(...args) {
```

Comment:
This function is very confusing.

Can you name the parameters explicitly and clearly? Make a distinction between date being submitted and actual date of submission.

Reply:
You can have more parameters in a function than parameters actually being passed. they will just become undefined.